CHANGELOG.md

```diff
@@ -0,0 +1,36 @@
+## CHANGELOG
+
+### 1.2.0 (2020-07-21)
+**SDK**
+- Upgraded build setup and some internal dependencies
+- Added support for customization of user agent information
+- Added feature to support backward & forward navigation support in a mini app
+- Added support for javascript of type module
+- Added feature to obtain geolocation data in a mini app
```

Comment:
Need to also mention that `requestPermission` was added and should be implemented by host app.

Reply:
This has a mention in the user guide at this moment. I think this will get better with the linking afterwards.